cds_ils/importer/providers/safari/ignore_fields.py

```diff
@@ -429,6 +429,10 @@
     "730__a",
     "7400_a",
     "740__a",
+    "758__4",
```

Comment:
did you clarify with Salomé if these fields should be ignored only for safari ?

Reply:
No I didn't, I will confirm and update for others if necessary.